src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -2058,6 +2058,9 @@ int AppleseedRenderer::Render(
     if (!m_rend_params.inMtlEdit || m_settings.m_log_material_editor_messages)
         create_log_window();
 
+    if (renderer_settings.m_noise_seed == 0)
```

Comment:
As we discussed, noise seed = 0 shouldn't be a special value. Automatic variation of noise seed per frame should be controlled with a checkbox.